src/Compilers/CSharp/Portable/Lowering/SyntheticBoundNodeFactory.cs

```diff
@@ -74,6 +74,12 @@ public MethodSymbol CurrentFunction
             }
         }
 
+        /// <summary>
+        /// Indicates whether we are inside a local function with an ExcludeCodeCoverageAttribute, possibly via multiple levels of nesting.
```

Comment:
>whether we are inside a local function with an ExcludeCodeCoverageAttribute [](start = 22, length = 75)

I think it would be better to not making this flag specifically about local functions, but rather define it more generally. We could mention the local function case as an example.